arches/app/etl_modules/jsonld_importer.py

```diff
@@ -99,6 +99,27 @@ def read(self, request):
                 ),
             }
 
+        try:
+            self.read_zip_file(content, result)
+        except zipfile.BadZipFile as e:
+            return {
+                "success": False,
+                "data": FileValidationError(
+                    message=e.args[0],
+                    code=400,
+                ),
+            }
```

Comment:
Noticed that in #10885 I forgot to surface this error in the UI where FILE_TYPE_CHECKING is false and we go straight to trying read a file as a zip file.